test/util/factories/message_factory.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:motis_mitfahr_app/util/chat/models/message.dart';
+
+import 'model_factory.dart';
+
+class MessageFactory extends ModelFactory<Message> {
+  @override
+  Message generateFake({
+    int? id,
+    DateTime? createdAt,
+    int? rideId,
```

Comment:
this is still missing the `ride` parameter to pass the whole object (see the other factories). Same for `sender`

Reply:
i don't have these Parameters in message since i don't need the hole object, so i guess it's fine

Reply:
That's really inconsistent, maybe we should just add those for completeness. Otherwise we will always be confused why they are not there

Reply:
i can do that but it's just more work for nothing

Reply:
I can do it

Reply:
did that